CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+##2015-02-17 - Supported Release 1.1.0
+### Summary
+
+Deprecates `type` in permissions array has been renamed to `perm_type`
+
+#### Features
+- Permissions parameter now takes array of hashes with `perm_type` instead of Puppet 4.0 protected word `type`
```

Comment:
No it still accepts 'type' it simply throws a warning.  We are scheduling a future feature release that will remove type entirely.